server/middlewares/stats.js

```diff
@@ -23,40 +16,82 @@ const isSameOrigin = (urlString1, urlString2) => {
 
 const tokenMatches = (token) => token === process.env.ADALITE_BACKEND_TOKEN
 
-const parseTxBodyOutAmount = (txBody) => {
+const decodeTxBody = (txBody) => {
+  try {
+    return cbor.decode(txBody)
+  } catch (e) {
+    // eslint-disable-next-line no-console
+    console.error(e)
+    return []
```

Comment:
this seems pretty dangerous, I would rather catch in `parseTxBodyOutAmount` and `parseTxBodyTotalAmount` and I would use `cbor.decode(txBody)` directly - extracting it to a function creates only an additional layer of indirection and saves a few characters at best

Reply:
fixed, please check

Reply:
the parsing functions now only have 1 line of code because of the change which seems way too thin but I kept them that way for documentation reasons (the comments attached to them)